src/dynamodb.rs

```diff
@@ -1,5 +1,6 @@
 use aws_config::retry::RetryConfig;
 use aws_sdk_dynamodb::model::AttributeValue;
+use aws_sdk_dynamodb::Client;
```

Comment:
```suggestion
#[cfg(feature = "test-utils")]
use aws_sdk_dynamodb::Client;
```

I think this would prevent the warnings about unused import.